go.mod

```diff
@@ -112,7 +112,6 @@ require (
 )
 
 replace (
-	github.com/RangelReale/osin => github.com/openshift/osin v1.0.1-0.20180202150137-2dc1b4316769
-	github.com/RangelReale/osincli => github.com/openshift/osincli v0.0.0-20160924135400-fababb0555f2
+	github.com/RangelReale/osin => github.com/openshift/osin v1.0.2-0.20220317075346-0f4d38c6e53f
```

Comment:
If you want, in your latest commit (`Updated osin to fix the uri validate issue `), you could fix the import of `osin` as well in an explicit form. Here's how I would go about it:

```shell
find pkg cmd -name '*.go' -exec sed -i 's|"github.com/RangelReale/osin"|"github.com/openshift/osin"|' {} +
go get github.com/openshift/osin@0f4d38c6e53f
go mod tidy && go mod vendor
```

Reply:
I think this definitely makes sense as the upstream repository is not maintained anymore and to the best of my knowledge our downstream fork is more or less the only one being maintained, cc @stlaz for verification.

Reply:
Generally though this change is independent from the contribution being done here.

Reply:
>this change is independent from the contribution being done here

Reopened #104. PTAL!

Reply:
(this comment is resolved, apologies for the noise!)